llvm/docs/LangRef.rst

```diff
@@ -8642,8 +8642,9 @@ The '``llvm.used``' Global Variable
 The ``@llvm.used`` global is an array which has
 :ref:`appending linkage <linkage_appending>`. This array contains a list of
 pointers to named global variables, functions and aliases which may optionally
-have a pointer cast formed of bitcast or getelementptr. For example, a legal
-use of it is:
+have a pointer cast formed of bitcast or getelementptr. The pointers are
+intentionally left unqualified to underline their ephemeral nature. For example,
+a legal use of it is:
```

Comment:
```suggestion
have a pointer cast formed of bitcast or getelementptr. The address space of the
pointers is always zero rather than the globals address space since these are not
real global references but rather a special marker for the code emission logic.
For example,
a legal use of it is:
```

Maybe something like this would be clearer?

Reply:
Thanks, this does sound much better so I've incorporated it, with the exception of explicitly calling out zero as the AS, because I still think it's not good to make it more magical than it already is. Please let me know if that works.